holidayextras/ui-toolkit
# Responsive images

#### What does this PR do? (please provide any background)
- Responsive images are here!
- This change allows you to specify a range of images for display at different widths using the `srcset` attribute.

#### What tests does this PR have?
None yet - this will not be merged

#### How can this be tested?
- Clone the UI Toolkit
- Run `grunt build`
- Run `grunt docs`
- When the docs have loaded up, go to the image component and view the responsive section (a cute panda)
- Shrink your browser window to a small size (smaller... smaller still) and hard refresh the page. You should see a green panda image with 1000w written on it.
- As you expand the window this image will change to blue (1500w) and red (2000w). This is swapping out images in a responsive manner.
- If you now shrink your browser back down to mobile, it will stay red (2000w) as that asset has already been downloaded.

if you need me to demo this, please ask :)

#### Screenshots / Screencast
<img width="349" alt="screen shot 2015-09-22 at 14 16 05" src="https://cloud.githubusercontent.com/assets/778942/10019299/7dbb2d22-6134-11e5-8dd1-57937f549696.png">
<img width="623" alt="screen shot 2015-09-22 at 14 15 10" src="https://cloud.githubusercontent.com/assets/778942/10019314/81b31d36-6134-11e5-9796-44e10c7b13fc.png">
<img width="684" alt="screen shot 2015-09-22 at 14 15 22" src="https://cloud.githubusercontent.com/assets/778942/10019317/849e35c6-6134-11e5-9de7-27837aaacbd6.png">


#### What gif best describes how you feel about this work?
![Panda](https://media.giphy.com/media/nntN4XMNwTzZ6/giphy.gif)

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
